nuxt.config.ts

```diff
@@ -21,6 +21,37 @@ export default defineNuxtConfig({
   app: {
     head: siteMeta,
   },
+  components: {
+    dirs: [
+      // Use this to find all the dirs `find . -name "*.vue" -exec dirname {} \; | sort -u`
+      { path: '~/components', sourcemap: true }, // Enable source map generation for components
```

Comment:
Maye let's keep this comment `// Enable source map generation for components` only once at the top, I guess no need to repeat on each line.

Reply:
Removed.